app/experimenter/nimbus-ui/src/components/FormMetrics/index.test.tsx

```diff
@@ -38,6 +38,15 @@ describe("FormMetrics", () => {
     await act(async () =>
       expect(screen.getByTestId("FormMetrics")).toBeInTheDocument(),
     );
+
+    expect(screen.getByTestId("tooltip-primary-probe-sets")).toHaveAttribute(
+      "data-tip",
+      "Specific metrics you’d like to impact in your experiment, which will be part of the analysis.",
+    );
+    expect(screen.getByTestId("tooltip-secondary-probe-sets")).toHaveAttribute(
+      "data-tip",
+      "Specific metrics that you are interested in observing in your experiment but they don't affect the results of your experiment.",
```

Comment:
given the length/specificity of these strings in two places it might make sense to constantize them, but i'm on the fence since it mean doing it for all tooltips

Reply:
That's a better idea, I'll go ahead and do that for these at least